packages/grant-explorer/src/features/ViewBallotPage.tsx

```diff
@@ -786,6 +789,12 @@ export default function ViewBallot() {
       setDonationError(newState);
     }
 
+    // check if wallet is connected
+    if (!address) {
```

Comment:
After adding this, the submit tests fails cause address is set to undefined even through we've mocked the  `useAccount` from wagmi in the tests